tests/br_restore_TDE_enable/run.sh

```diff
@@ -0,0 +1,125 @@
+#!/bin/bash
+#
+# Copyright 2020 PingCAP, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eux
+DB="$TEST_NAME"
+TABLE="usertable"
+DB_COUNT=1
+
+# start Minio KMS service
+
+export MINIO_KMS_KES_ENDPOINT=https://play.min.io:7373
```

Comment:
> The KES instance at `https://play.min.io:7373` is meant to experiment and provides a way to get started quickly.
> Note that anyone can access or delete master keys at `https://play.min.io:7373`. You should run your own KES instance in production.

is it possible the KMS become unavailable 🤔

Reply:
I have no idea, and I had built a KES instance based on filesystem keystore, the backup process will cause the following problem:
```
2021/07/08 17:25:23 http: TLS handshake error from 127.0.0.1:50530: remote error: tls: bad certificate

API: PutObject(bucket=mybucket, object=br_restore_TDE_enable/backup.lock)
Time: 17:25:23 CST 07/08/2021
DeploymentID: 62ea50fd-5f6e-4148-83b1-4a5d44dd14f2
RequestID: 168FC69289933DF3
RemoteHost: 127.0.0.1
Host: 127.0.0.1:24927
UserAgent: aws-sdk-go/1.35.3 (go1.16.5; linux; amd64)
Error: Post https://127.0.0.1:7391/v1/key/generate/my-minio-key: x509: certificate signed by unknown authority
       4: cmd/api-errors.go:1786:cmd.toAPIErrorCode()
       3: cmd/api-errors.go:1811:cmd.toAPIError()
       2: cmd/object-handlers.go:1360:cmd.objectAPIHandlers.PutObjectHandler()
       1: net/http/server.go:2007:http.HandlerFunc.ServeHTTP()

```

Reply:
@zwj-coder is the KES server configured like https://github.com/minio/kes/wiki/Getting-Started#22-create-the-root-identity?

Reply:
It seems that the tikv node have not certification to access KES server.

Reply:
@kennytm Yes

Reply:
@kennytm I had replaced outside instance `https://play.min.io:7373` with a self-made KES Server , the previous problem was caused by that KES Server not trust the self-signed certification, and we should explicitly set 
`export MINIO_KMS_KES_CA_PATH=<CA-or-KES-server-certificate>`